hadoop-tools/hadoop-gcp/pom.xml

```diff
@@ -351,7 +351,6 @@
                   </shadedPattern>
                 </relocation>
               </relocations>
-              <shadedArtifactAttached>true</shadedArtifactAttached>
```

Comment:
Why are we removing this?

Reply:
We have two options:

1. shadedArtifactAttached=true means a separate shaded jar artifact will be attached to the build. Clients can depend on it by specifying `<classifier>shaded</classifier>` in the dependency declaration.
2. shadedArtifactAttached=false (the default, hence removed in this patch) means the shaded jar completely replaces the published artifact. I think this is what we want for hadoop-gcp, given the tight coupling to specific shaded versions of things like Guava and gRPC. There wouldn't be much value in the non-shaded artifacts.